### ISSUE TITLE
_Open_TabToNextWindow

### ISSUE THREAD

**Issue by GiancarloPozzoCoEPP** [rshev/MoveTabToNextWindow#7]
This is a suggestion for a similar Firefox Addons.
The current addon moves a tab that is _already opened_. I'd like to _open a new tab_ in the next window with a shift+click (or other modifier key).

The great advantage is while using 2 monitors:
eg. I'm reading the wikipedia webpage about Italy on my left monitor. I want to quickly open (shift+click) the link to Rome in a previous opened window that stays in the right monitor.
I don't want to wait that the Rome webpage is loaded, or focus on another tab and move it. I want to continue reading the Italy webpage on my left monitor. When the webpage on the right monitor is loaded, I can look at it.

Thank you for your time, I really appreciate your work :)

**Comment by rshev** [comment]
I'm not sure what you describe is possible with current webextension APIs. However I'll keep an eye out.
Did you know that you can set up a hotkey for MoveTabToNextWindow? Maybe that'll help somehow.

**Comment by GiancarloPozzoCoEPP** [comment]
Yes, I know about the hotkey. The problem is that I need to open a new tab and focus on this tab (this breaks my workflow), then I can send it to the other window with the hotkey, then go back to the original page and continue reading/working there. What I'm looking for is just a shift+click to open the link in the other window. Anyway, thank you for your reply :)
